packages/trace-viewer/src/ui/networkResourceDetails.css

```diff
@@ -19,34 +19,32 @@
   height: 100%;
   user-select: text;
   line-height: 24px;
-  margin-left: 10px;
   overflow: auto;
-}
 
-.network-request-details-url {
-  white-space: normal;
-  word-wrap: break-word;
-  margin-left: 10px;
-}
+  em {
+    margin: 10px
+  }
 
-.network-request-details-headers {
-  white-space: pre;
-  overflow: hidden;
-  margin-left: 10px;
-}
+  .network-details {
```

Comment:
Probably should scope this class name better, as it's more generic than we typically have.